### DISCUSSION TITLE
Avatar for SpeedNetworking Bot

### DISCUSSION THREAD

**Discussion by TakshakRamteke** [EddieHubCommunity/EddieBotBreakout#11] [General]
Hey people :wave:  i just created a rough doodle for the avatar, would love your suggestions
also this is not complete it needs a bit of shading and all 
![IMG20211029225745](https://user-images.githubusercontent.com/44837950/139479267-4d104e66-6b31-4ac1-b68c-696095fd008c.jpg)

**Comment by eddiejaoude** [comment]
Looks great, probably best to keep on brand and use something related to our existing community avatar https://github.com/EddieHubCommunity/Branding
